dev/com.ibm.ws.cdi.beansxml_fat/fat/src/com/ibm/ws/cdi12/fat/tests/BeanDiscoveryModeNoneTest.java

```diff
@@ -56,19 +63,25 @@ public static Archive buildShrinkWrap() {
      * @see com.ibm.ws.fat.LoggingTest#getSharedServer()
      */
     @Override
-    protected ShrinkWrapSharedServer getSharedServer() {
-        return SHARED_SERVER;
+    protected SharedServer getSharedServer() {
+        return null;
     }
 
     @Test
     @ExpectedFFDC({ "javax.servlet.UnavailableException", "com.ibm.wsspi.injectionengine.InjectionException" })
     public void testcorrectExceptionThrown() throws Exception {
-        this.verifyStatusCode("/beanDiscoveryModeNone/TestServlet", 404);
+        WebBrowser webBrowser = WebBrowserFactory.getInstance().createWebBrowser((File) null);
+        int statusCode = 404;
+        int actualStatusCode = webBrowser.request(createURL("/beanDiscoveryModeNone/TestServlet"), statusCode);
+
+        if (actualStatusCode != statusCode) {
+            throw new WebBrowserException("The expected status code was not thrown.  Expected " + statusCode + ", was " + actualStatusCode);
+        }
     }
 
     @AfterClass
     public static void tearDown() throws Exception {
-        if (SHARED_SERVER != null && SHARED_SERVER.getLibertyServer().isStarted()) {
+        if (server != null && server.isStarted()) {
```

Comment:
I don't think you need this `if` block (`server` will never be `null` and if the server is not started then `stopServer()` will just print a message and return)